File: Resource/51900286-51900712-51900718-51900699/source demo/Window Form/51900286-51900712-51900718-51900699/RestaurentDTO/PhieuGoiMon.cs
using System.Data;

namespace RestaurentDTO
{
    public class PhieuGoiMon
    {
        private string _maOrder;
        private string _ghiChuMonAn;
        private string _maBanAn;
        private string _maNhanVien;

        public PhieuGoiMon(string maOrder, string ghiChuMonAn, string maBanAn, string maNhanVien)
        {
            _maOrder = maOrder;
            _ghiChuMonAn = ghiChuMonAn;
            _maBanAn = maBanAn;
            _maNhanVien = maNhanVien;
        }

        public PhieuGoiMon(DataRow dataRow)
        {
            _maOrder = dataRow["MaOrder"].ToString();
            _ghiChuMonAn = dataRow["GhiChuMonAn"].ToString();
            _maBanAn = dataRow["MaBanAn"].ToString();
            _maNhanVien = dataRow["MaNhanVien"].ToString();
        }

        public string MaOrder { get => _maOrder; set => _maOrder = value; }
        public string GhiChuMonAn { get => _ghiChuMonAn; set => _ghiChuMonAn = value; }
        public string MaBanAn { get => _maBanAn; set => _maBanAn = value; }
        public string MaNhanVien { get => _maNhanVien; set => _maNhanVien = value; }
    }
}

File: Dev/Models/Models/PhieuGoiMon.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="phieugoimon")
public  class PhieuGoiMon {
    @Id
    @GeneratedValue
    private Long MaOrder;
    
    private String GhiChuMonAn;
    private Long MaBanAn;
    private Long MaNhanVien;

    @OneToMany(mappedBy = "phieugoimon")
    Set<Chitietgoimon> chitietgoimons;
}